pages/concepts/drop-in-components.mdx

```diff
@@ -46,21 +52,189 @@ Dwolla’s Drop-in UI Components are customizable to match the look and feel of
 
 ## Supported Components
 
-Components are namespaced with `dwolla-` followed by the component name. For example:
+Dwolla's UI components library contains a variety of supported components that represent isolated functions or connected flows. Each component requires a unique “client-token” and Customer ID for the end-user that is performing the action in your web application. This section outlines the complete list of supported components which includes: the component name, css classes for customization, and the required client-token action.
 
-`<dwolla-document-upload customerId="{{ customer.id }}"></dwolla-document-upload>`
+<InlineCTA
+  icon={guideIcon}
+  text="For more information on integrating drop-in components, reference our Guide which walks through how to use drop-in components in full detail."
+  href="/guides/drop-ins/"
+/>
+
+### Create an Unverified Customer
+
+The `dwolla-customer-create` component is used to display a form to collect the required information needed to create an Unverified Customer. An Unverified Customer is a customer type that requires a minimal amount of information in order to be created and send funds. This information includes: First Name, Last Name, Email, Optionally: Business Name for Businesses. To find out more about the abilities and limitations of this customer type, [visit our docs](/concepts/customer-types) on concepts.
+
+[Create client-token](https://docs.dwolla.com/#create-a-client-token) action: `customer.create`
+
+##### CSS Classes for Create an Unverified Customer
+
+```css
+/* Available css classes for customization*/
+dwolla-customer-create,
+dwolla-input-container,
+dwolla-customer-input,
+dwolla-customer-firstName,
+dwolla-customer-lastName,
+dwolla-customer-email,
+dwolla-customer-tos,
+dwolla-customer-checkbox,
+dwolla-customer-text,
+dwolla-submit,
+dwolla-customer-submit,
+dwolla-success,
+dwolla-success-message,
+dwolla-error,
+dwolla-error-message
+```
+
+##### Example Create an Unverified Customer UI
+
+<Image
+  src={createCustomerDropIn}
+  alt="Create a Customer Drop-in Component"
+  width="75%"
+  height="75%"
+/>
+
+### Upgrade an Unverified Customer
+
+The `dwolla-customer-update` component is used to upgrade an Unverified Customer into a Personal Verified Customer. Upgrading an Unverified Customer to a Verified Customer will give them the ability to increase their transaction limits. For more information on the difference between an Unverified and Verified Customer, visit our docs on concepts.
```

Comment:
@misskellymore , could we add a link here for [visit our docs](/concepts/customer-types/) ?

Reply:
yes